### PR TITLE
Fixes #10102: Package datasource plugin properly

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/10102

### CODE REVIEW

**Comment by peckpeck** [comment]
I will let both @amousset and @fanf look at this one.

**Comment by amousset** [comment]
@fanf @peckpeck  OK for me.

**Comment by peckpeck** [comment]
You may wait for the package manager to be merge first. To make sure this works.

**Comment by fanf** [comment]
It is ok for me, to, sorry forgot to tell.

**Comment by amousset** [comment]
@peckpeck Do you want to test it again before merging?

File: packaging/metadata [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "type": "plugin",
+  "name": "${plugin-id}",
+  "version": "${plugin-version}",
+  "build-date": "${maven.build.timestamp}",
+  "build-commit": "${commit-id}",
+  "jar-files": [ "datasources.jar" ],
```
Comment (amousset) [inline packaging/metadata]: @peckpeck  This now needs to be a full path

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**